### PR TITLE
fix copy 0 size file on windows

### PR DESCRIPTION
**Pull request by 2fly2**
problem:
wfreerdp connect to windows 10, copy file from remote, paste local, get a 0 size file.

root cause:
rdp server(remote windows 10) tells wfreerdp client file size, but we not use it. see flag FD_FILESIZE.

fix:
when server send file size, then  init file size with it.

### CODE REVIEW

File: client/Windows/wf_cliprdr.c [hunk 1]
```diff
@@ -1482,15 +1486,12 @@ static LRESULT CALLBACK cliprdr_proc(HWND hWnd, UINT Msg, WPARAM wParam, LPARAM
 				case OLE_SETCLIPBOARD:
 					DEBUG_CLIPRDR("info: OLE_SETCLIPBOARD");
 
-					if (S_FALSE == OleIsCurrentClipboard(clipboard->data_obj))
```
Comment (akallabeth) [inline client/Windows/wf_cliprdr.c:1485]: Why is this line removed?

Comment (2fly2) [reply client/Windows/wf_cliprdr.c:1485]: when copy file A.txt from remote, then copy file B.txt, if add this line, will not place file B.txt in clipboard, because file A.txt clipboard data object is still in clipboard, so  OleIsCurrentClipboard(clipboard->data_obj) return S_OK.

**Comment by akallabeth** [comment]
@freerdp-bot test

**Review by akallabeth — APPROVED** [review]
